pom.xml

```diff
@@ -2891,6 +2891,13 @@
               <arg>-Wconf:cat=unchecked&amp;msg=outer reference:s</arg>
               <arg>-Wconf:cat=unchecked&amp;msg=eliminated by erasure:s</arg>
               <arg>-Wconf:msg=^(?=.*?a value of type)(?=.*?cannot also be).+$:s</arg>
+
```

Comment:
nit. Redundant empty line.

Reply:
Thanks @dongjoon-hyun , [e80f584](https://github.com/apache/spark/pull/41356/commits/e80f584da7ee5beff27ca9209e81488aa285c3c0) removed it